examples/test_spectrum_model.py

```diff
@@ -1,47 +1,74 @@
 """Spectrum model API for Gammapy.
 """
+import numpy as np
 from astropy.units import Quantity as Q
 from gammapy.spectrum.models import (
     SpectralModel, PowerLaw, ExponentialCutoffPowerLaw
 )
+from gammapy.irf import (
+    EffectiveAreaTable,
+    EnergyDispersion
+)
+from gammapy.utils.energy import EnergyBounds
+
+
+# TODO clean up and add to gammapy.irf
```

Comment:
Yes, I did. but the ``from_gauss`` methods also needs to be tested/extended to nxm and I wanted to do that in a separate PR.